content/articles/testing-for-broken-authentication-in-web-apps/index.md

```diff
@@ -0,0 +1,117 @@
+Authentication is a mechanism put in place to determine if a user is who they say they are, through either a username and password-based system or any other form of authentication system.
+
+### What is Broken Authentication?
+
+Broken Authentication is a type of vulnerability that allows attackers to get into a web application without proper credentials. This could be carried out either by bypassing the authentication mechanism put in place or brute=forcing another user's account. If the attacker successfully bypasses or brute-forces his way into another user's account, they gain access to all the data and privileges of that user account.
+
+According to the [OWASP Top 10 2021 report](https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/), Broken Authentication is ranked number 7 and grouped under `Identification and Authentication Failures`. 
+
+The severity of this vulnerability can be so high. Say, an attacker was able to brute-force his way into the administrator account of a web application, this means he gets full control over the web application. 
+
+This article seeks to demonstrate how an attacker tests for broken authentication in a web application and how to prevent them.
+
+> **Disclaimer:** This article is for educational purposes only.
+
+### Testing for Broken Authentication
+To carry out this attack, I'll be making use of an intentionally vulnerable web application called [Juice shop](https://github.com/juice-shop/juice-shop).
```

Comment:
Do you run this locally? If so, brief a little about installation steps and setting up, before you get started with the actual stuff.

If you are not going to use local, then make sure to specify this hosted URL - http://demo.owasp-juice.shop/#/login